### DISCUSSION TITLE
expo-camera is under-documented

### DISCUSSION THREAD

**Discussion by maxkrieger** [expo/expo#25123] [Proposals and Ideas]
The [usage section here](https://docs.expo.dev/versions/latest/sdk/camera/#usage) has no info about how to actually take a picture! It appears you need to create a ref but there's no example of that.

**Comment by amandeepmittal** [comment]
Hey @maxkrieger, I agree. The example should be improved and at least cover the basic functionality of the library but at the same time is minimal. Apologies for the inconvenience so far, we'll update this example.

**Reply by maxkrieger** [reply]
As a side note, the styles in the example should also be fleshed out because when the height of the `<Camera>` element is 0, the native side throws a vague error about failing to save the image.

**Reply by amandeepmittal** [reply]
Nice catch! Thanks for letting us know about the styles.
